client/state/followers/actions.js

```diff
@@ -19,38 +10,82 @@ import {
 	FOLLOWER_REMOVE_SUCCESS,
 } from 'calypso/state/action-types';
 
-export default {
-	fetchFollowers( query, silentUpdate = false ) {
-		return ( dispatch ) => {
-			// TODO: Componentes should not fetch if already fetching
-			debug( 'fetching followers', query );
-			if ( ! silentUpdate ) {
-				dispatch( {
-					type: FOLLOWERS_REQUEST,
-					query,
-				} );
-			}
-			wpcom
-				.site( query.siteId )
-				.statsFollowers( query )
-				.then( ( data ) => dispatch( { type: FOLLOWERS_RECEIVE, query, data } ) )
-				.catch( ( error ) => dispatch( { type: FOLLOWERS_REQUEST_ERROR, query, error } ) );
-		};
-	},
-	removeFollower( siteId, follower ) {
-		return ( dispatch ) => {
-			debug( 'removing follower', follower, siteId );
-			dispatch( {
-				type: FOLLOWER_REMOVE_REQUEST,
-				siteId: siteId,
-				follower: follower,
-			} );
-			wpcom
-				.undocumented()
-				.site( siteId )
-				.removeFollower( follower.ID )
-				.then( ( data ) => dispatch( { type: FOLLOWER_REMOVE_SUCCESS, siteId, follower, data } ) )
-				.catch( ( error ) => dispatch( { type: FOLLOWER_REMOVE_ERROR, siteId, follower, error } ) );
-		};
-	},
-};
+/**
+ * Returns an action object used in signalling that followers for the site
+ * have been requested..
+ *
+ * @param {object} query params to be sent to the API
+ */
+export const requestFollowers = ( query ) => ( {
+	type: FOLLOWERS_REQUEST,
+	query,
+} );
+
+/**
+ * Returns an action object used in signalling that a request for followers
+ * was successfull.
+ *
+ * @param {object} query params which were sent to the API
+ * @param {object} data API response object
+ */
+export const successRequestFollowers = ( query, data ) => ( {
```

Comment:
Naming nit: I'd rename this to `requestFollowersSuccess`.

Reply:
done via 3b3bfc8